cucumber/authenticators_jwt/features/authn_jwt_check_standard_claims.feature

```diff
@@ -0,0 +1,367 @@
+Feature: JWT Authenticator - Check standard claim
+
+  Background:
+    Given I initialize JWKS endpoint with file "myJWKs.json"
+    And I load a policy:
+    """
+    - !policy
+      id: conjur/authn-jwt/raw
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on raw JWKs.
+
+      - !variable
+        id: token-app-property
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !policy
+      id: conjur/authn-jwt/keycloak
+      body:
+      - !webservice
+        annotations:
+          description: Authentication service for JWT tokens, based on Keycloak as OIDC provider.
+
+      - !variable
+        id: provider-uri
+
+      - !variable
+        id: token-app-property
+
+      - !variable
+        id: issuer
+
+      - !group users
+
+      - !permit
+        role: !group users
+        privilege: [ read, authenticate ]
+        resource: !webservice
+
+    - !host
+      id: alice
+      annotations:
+        authn-jwt/keycloak/email: alice@conjur.net
+
+    - !grant
+      role: !group conjur/authn-jwt/keycloak/users
+      member: !host alice
+
+    - !host
+      id: myapp
+      annotations:
+        authn-jwt/raw/project-id: myproject
+
+    - !grant
+      role: !group conjur/authn-jwt/raw/users
+      member: !host myapp
+    """
+    And I am the super-user
+    And I successfully set authn-jwt "token-app-property" variable to value "host"
+    And I have a "variable" resource called "test-variable"
+    And I add the secret value "test-secret" to the resource "cucumber:variable:test-variable"
+    And I permit host "myapp" to "execute" it
+    And I permit host "alice" to "execute" it
+
+  Scenario: Issuer configured with incorrect value, iss claim not exists in token, 200 ok
```

Comment:
most of your tests not include iss claim, what is the different here?

Reply:
In most of test we check other things. iss is in flow of checking standard claims so its enough to test it here